test/sanity.js

```diff
@@ -187,3 +187,25 @@ transformFile('test/fixtures/test-export-all-as.jsx', {
   if (err) throw err;
   console.log('test/fixtures/test-export-all-as.jsx', result.code);
 });
+
+transformFile('test/fixtures/test-commented.jsx', {
+  presets: ['airbnb'],
+  plugins: [
+    inlineReactSvgPlugin,
+  ],
+}, (err, result) => {
+  if (err) throw err;
+  console.log('test/fixtures/test-commented.jsx', result.code);
+});
+
+/* TODO: uncomment if babel fixes its parsing for SVGs
```

Comment:
It would be good to link to an upstream issue here.

Reply:
There isn’t one filed yet; I’m deferring to the issue reporter to do so.